source/speechViewer.py

```diff
@@ -96,7 +133,16 @@ def appendText(text):
 	#Otherwise it would be impossible to select text, or even just read it (as a blind person).
 	if _guiFrame.FindFocus()==_guiFrame.textCtrl:
 		return
-	_guiFrame.textCtrl.AppendText(text + "\n")

```

Comment:
This way of showing line numbers worries me. If NVDA's line number reporting is on, it is most likely that line numbers are double spoken.

I believe [wx.stc.StyledTextCtrl](https://wxpython.org/Phoenix/docs/html/wx.stc.StyledTextCtrl.html) supports line numbers and it should be accessible as well. It would be great if you could investigate the use of that control. However, I"m afraid it will also change the way how you should set the font settings on the control.

Alternatively, I'd prefer leaving out the line numbers for now.